server.js

```diff
@@ -47,10 +47,7 @@ if ('production' === process.env.NODE_ENV || 'test' === process.env.NODE_ENV) {
       }
 
       const document = JSON.parse(data);
-
-      if (document.readonly) {
-        delete(document.readonly);
-      }
+      document.readonly = !!document.readonly;
```

Comment:
I don't get the point of this. I am sure there is some kind of JavaScript madness you can explain to me 😛